tests/product-cart.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+    await page.goto(process.env.WEB_URL as string);
+    await page.click('[data-test="product-01JR5X9HFW34R5EZ9D2SZ1H1GR"]');
+});
+
+test.describe('Verify user can add product to the cart', () => {
+    test('Verify user can navigate to the product page', async ({ page }) => {
+        
+        await expect(page.locator('[data-test="product-name"]')).toHaveText('Slip Joint Pliers');
```

Comment:
I'd use the getByTestId method to find an element. But you must do this before https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute